.travis.yml

```diff
@@ -44,7 +46,7 @@ before_script:
 # Install composer dependencies
   - composer validate
   - composer require --no-update silverstripe/recipe-cms:4.3.x-dev
```

Comment:
We may have to change  `recipe-cms:4.3.x-dev` to `^4` in that case. Otherwise, it's still going to pull `4.3.x-dev`

Reply:
I'm going to start a separate RFC about that I think ;-)